### PR TITLE
Adding Fgbio FilterConsensusReads

### LINKED ISSUE #58: Write CWL and Docker for Fgbio FilterConsensusReads
Tool: http://fulcrumgenomics.github.io/fgbio/tools/latest/FilterConsensusReads.html
Version: https://github.com/fulcrumgenomics/fgbio/releases/tag/1.2.0

Example Command:

Duplex BAM
```bash
/opt/common/CentOS_7/java/jdk1.8.0_202/bin/java -Djava.io.tmpdir=/scratch/ -Xmx4g -jar /home/arorak/repositories/fgbio/target/scala-2.13/fgbio-1.2.0-c8e1c41-SNAPSHOT.jar FilterConsensusReads --input /juno/work/bergerm1/bergerlab/arorak/Project_benchmark_fgbio/FqToBamFgbio_2020-06-10/NovaSeq_Plasma_Healthy_Donor/DONOR11-N/DONOR11-N.fqtobam.clip.map.md.abra2.fm.group.cons.map.abra2.fm.bam --output /juno/work/bergerm1/bergerlab/arorak/Project_benchmark_fgbio/FqToBamFgbio_2020-06-10/NovaSeq_Plasma_Healthy_Donor/DONOR11-N/DONOR11-N.fqtobam.clip.map.md.abra2.fm.group.cons.map.abra2.fm.duplex.bam -r /work/access/production/resources/reference/versions/hg19/Homo_sapiens_assembly19.fasta -M=2 1 1 -N=30 --reverse-per-base-tags=true
```
Simplex + Duplex :

```bash
/opt/common/CentOS_7/java/jdk1.8.0_202/bin/java -Djava.io.tmpdir=/scratch/ -Xmx4g -jar /home/arorak/repositories/fgbio/target/scala-2.13/fgbio-1.2.0-c8e1c41-SNAPSHOT.jar FilterConsensusReads --input /juno/work/bergerm1/bergerlab/arorak/Project_benchmark_fgbio/FqToBamFgbio_2020-06-10/NovaSeq_Plasma_Healthy_Donor/DONOR11-N/DONOR11-N.fqtobam.clip.map.md.abra2.fm.group.cons.map.abra2.fm.bam --output /juno/work/bergerm1/bergerlab/arorak/Project_benchmark_fgbio/FqToBamFgbio_2020-06-10/NovaSeq_Plasma_Healthy_Donor/DONOR11-N/DONOR11-N.fqtobam.clip.map.md.abra2.fm.group.cons.map.abra2.fm.simplex.pre.bam -r /work/access/production/resources/reference/versions/hg19/Homo_sapiens_assembly19.fasta -M=3 3 0 -N=30 --reverse-per-base-tags=true
```
- [x] CWL
- [x] Docker
- [ ] Test with cwltool/toil
- [x] Example input
- [x] Readme

### PR DESCRIPTION
**Pull request by rhshah**
:heavy_check_mark: CWL
:heavy_check_mark: Readme
:heavy_check_mark: example yaml
:heavy_check_mark: test with cwltool of help
:heavy_check_mark: Update on summary.md

Resolves #58

### CODE REVIEW

**Comment by ionox0** [comment]
looks good to me

File: fgbio_filter_consensus_reads_1.2.0/fgbio_filter_consensus_reads_1.2.0.cwl [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+      prefix: '--min-base-quality'
+    doc: Mask (make N) consensus bases with quality less than this threshold.
+  - id: max_no_call_fraction
+    type: float?
+    inputBinding:
+      position: 0
+      prefix: '--max-no-call-fraction'
+    doc: Maximum fraction of no-calls in the read after filtering
+  - id: min_mean_base_quality
+    type: File?
```
Comment (murphycj2) [inline fgbio_filter_consensus_reads_1.2.0/fgbio_filter_consensus_reads_1.2.0.cwl:83]: should the type be int?

Comment (rhshah) [reply fgbio_filter_consensus_reads_1.2.0/fgbio_filter_consensus_reads_1.2.0.cwl:83]: Good Catch that is correct

File: fgbio_filter_consensus_reads_1.2.0/fgbio_filter_consensus_reads_1.2.0.cwl [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  - id: min_mean_base_quality
+    type: int?
+    inputBinding:
+      position: 0
+      prefix: '--min-mean-base-quality'
+    doc: The minimum mean base quality across the consensus read
+  - id: require_single_strand_agreement
+    type: boolean?
+    inputBinding:
+      position: 0
```
Comment (ionox0) [inline fgbio_filter_consensus_reads_1.2.0/fgbio_filter_consensus_reads_1.2.0.cwl:91]: I'm not sure if this is Rabix automatically adding the `position: 0` field for every input, but technically we don't need that if we have `prefix: `.

**Pull request merged**